src/App.js

```diff
@@ -1,17 +1,130 @@
-import './App.css';
+/* eslint-disable no-console */
+/* eslint-disable react/jsx-props-no-multi-spaces */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable max-classes-per-file */
 import React from 'react';
+import { Dropdown, Option } from './Dropdown';
 
-function App() {
-  // fetches JSON data passed in by flask.render_template and loaded
-  // in public/index.html in the script with id "data"
-  // const args = JSON.parse(document.getElementById("data").text);
-
-  // This main page will be further designed with react components
-  // so the user can interact with the main app page. Some Data will also
-  // be present for viewing when user lands on this page.
-  return (
-    <h1> ! Welcome To Main App Page User !</h1>
-  );
+// This component will handle the diet lookup feature which will direct user to
+// the best overall Diets.
+class App extends React.Component {
```

Comment:
Why are you using a class component instead of function-style here? I think this could definitely be refactored into a function component.

Reply:
I am still not too good with reactJS, but I will try and refactor this entire file to break up the functions and do a function style component.